cognite/client/data_classes/data_modeling/ids.py

```diff
@@ -148,7 +151,17 @@ class DataModelId(VersionedDataModelingId):
 NodeIdentifier = Union[NodeId, Tuple[str, str, str]]
 EdgeIdentifier = Union[EdgeId, Tuple[str, str, str]]
 
-Id = Union[Tuple[str, str], Tuple[str, str, str], DataModelingId, VersionedDataModelingId, NodeId, EdgeId, InstanceId]
+Id = Union[
+    Tuple[str, str],
+    Tuple[str, str, str],
+    DataModelingId,
+    VersionedDataModelingId,
+    "View",
```

Comment:
This is the controversial `mypy` fix? I do not think I like treating `View` as an Id, as it is not a Id. Then, prefer just to a `ignore`.

Reply:
See suggestion in the `_apl/instances.py`

Reply:
Reverted